sdks/java/io/kafka/pom.xml

```diff
@@ -40,6 +40,35 @@
             <skip>true</skip>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <executions>
+            <execution>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <artifactSet>
+                  <includes>
+                    <include>com.google.guava:guava</include>
+                  </includes>
+                </artifactSet>
+                <relocations>
```

Comment:
Random comment, nothing to do now:

I worry that in a future version a shaded dependency will fall out of the relocations specified here, and we won't know. If there's an easy ability to "assert" that the final jar contains nothing outside `org.apache.beam.<component>` would be awesome.